src/main/java/cst438flights/controller/flightRestController.java

```diff
@@ -46,13 +46,40 @@
         }
     }
 
-    @GetMapping("/api/flights/{departureAirport}/{arrivalAirport}")
+    @GetMapping(value = {"/api/flights/{departureAirport}/{arrivalAirport}"})
     public ResponseEntity<List<Flight>> getAvailableFlights(
             @PathVariable("departureAirport") String departureAirport,
-            @PathVariable("arrivalAirport") String arrivalAirport
+            @PathVariable("arrivalAirport") String arrivalAirport,
+            String date
             ) {
+        if(date == null) {
+            date = "";
+        }
+
+        List<Flight> flightInfo = flightService.getAvailableFights(departureAirport, arrivalAirport, date);
+        System.out.println(flightInfo);
+
+        if (flightInfo == null) {
+            return new ResponseEntity<List<Flight>>(HttpStatus.NOT_FOUND);
+        } else {
+            return new ResponseEntity<List<Flight>>(flightInfo, HttpStatus.OK);
+        }
+    }
+    
```

Comment:
Added the extra API option that allows the planner to enter a date. Andrew, thanks for the idea!